repose-aggregator/components/filters/translation/pom.xml

```diff
@@ -87,11 +86,6 @@
             <artifactId>xalan</artifactId>
             <version>2.7.1</version>
         </dependency>
-        <dependency>
```

Comment:
I understand that.

I lied anyway, though. It's actually dependency managed in a parent pom, but the dependency is duplicated in this pom. So I just removed the duplicate dependency.